### ISSUE TITLE
Brownie test subcommand

### ISSUE THREAD

**Issue by fubuloubu** [iamdefinitelyahuman/brownie#302]
### What was wrong?
Have a new subcommand `brownie test` that just delegates to `pytest tests/`. Might make it easier for Brownie n00bs who haven't used pytest before.

**Comment by iamdefinitelyahuman** [comment]
I really like this idea, but my first attempt at implementing it keeps coming up against warnings like this one:

```
_pytest/config/__init__.py:890: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: brownie
    self._mark_plugins_for_rewrite(hook)
```

Because of it, even with a fully passing test suite the final report shows `>0` errors and is printed in yellow.  Not very friendly for new users.

Suppressing the warning with `-W` or the `warnings` library doesn't seem to work. I'm sure there's another way - hijack the final report and remove those warnings, or launch pytest as a subprocess maybe.

Will keep working on it.

**Comment by iamdefinitelyahuman** [comment]
Implemented in 6a0571e204b10d6733d4207457adc75368252617 - still needs test cases but it seems to be working.  I managed to silence the warnings through a `pytest_sessionstart` hook.

Closing this issue for now, it's still active in #295 until [`iamdefinitelyahuman:pytest-updates`](https://github.com/iamdefinitelyahuman/brownie/tree/pytest-updates) merges.
